src/test/java/Domashka17.java

```diff
@@ -0,0 +1,42 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Domashka17 extends BaseTest {
+
+
+    @Test
+    public static void addSongToThePlaylist() {
+        BaseTest B = new BaseTest();
+        B.setupBrowser();
+        B.navigation("https://bbb.testpro.io/");
+        B.enterEmail("valentineguts@gmail.com");
+        B.enterPassword("te$t$tudent");
+        B.submit();
+        B.search("hozlinaCC0");
+       // B.sleep();
+        B.clickViewALl();
+        B.clickFirstSong();
+        B.AddTo();
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example **addTo()**